src/frontend/src/views/Data/index.tsx

```diff
@@ -74,7 +74,7 @@ const Data: FunctionComponent = () => {
     }
     const transformedData = category.data.map((datum: BioinformaticsData) => {
       const transformedDatum = Object.assign({}, datum);
-      category.transforms!.forEach((transform) => {
+      category.transforms?.forEach((transform) => {
```

Comment:
You can leave this as `!` asserted defined, since we check for the undefined case immediately beforehand

Reply:
Oh thanks! Put it back in and added `eslint-disable-next-line` since `eslint` is throwing an error about this incorrectly 👍